app/src/features/Batches/BatchList.tsx

```diff
@@ -136,49 +140,50 @@ const BatchList = (): JSX.Element => {
       <div className={classes.accordions}>
         {batches?.results &&
           batches.results.map((batch) => (
-            <Accordion key={`batch-${batch.id}`} className={classes.accordion}>
+            <Accordion className={classes.accordion} key={`batch-${batch.id}`}>
               <AccordionSummary
-                className={classes.accordionSummary}
                 expandIcon={<ExpandMore />}
                 aria-controls="panel1a-content"
                 id="panel1a-header"
+                classes={{
+                  root: classes.accordionSummary,
+                  content: classes.accordionSummaryContent,
+                }}
               >
-                <div className={classes.accordionSummaryContent}>
-                  <div className={classes.accordionSummaryTitle}>
-                    <div>
-                      <Typography variant="subtitle2">
-                        {!batch.deleted_at && `⏳ In Progress...`}
-                        {batch.deleted_at &&
-                          batch.errors.length > 0 &&
-                          `⚠️ ${batch.errors.length} errors`}
-                        {batch.deleted_at &&
-                          batch.errors.length === 0 &&
-                          `✅ Success`}
-                      </Typography>
-                      <Typography className={classes.batchId}>
-                        {batch.id}
-                      </Typography>
-                    </div>
+                <div className={classes.accordionSummaryTitle}>
+                  <div>
                     <Typography variant="subtitle2">
-                      {new Date(batch.created_at)
-                        .toLocaleString()
-                        .split(",")
-                        .join(" -")}
+                      {!batch.deleted_at && `⏳ In Progress...`}
+                      {batch.deleted_at &&
+                        batch.errors.length > 0 &&
+                        `⚠️ ${batch.errors.length} errors`}
```

Comment:
Try using the translate `t` function

Maybe it is possible to set emoticons inside string values ?

Reply:
it is !